src/utils/annotation.rs

```diff
@@ -0,0 +1,132 @@
+use tree_sitter::{Parser, Query, QueryCursor};
+
+#[allow(dead_code)]
+pub fn read_vault_annotation(src: &str) -> Vec<(&str, &str)> {
+    let elixir_lang = tree_sitter_elixir::language();
+    let mut parser = Parser::new();
+    parser.set_language(elixir_lang).unwrap();
+
+    let tree = parser.parse(src, None).unwrap();
+    println!("{}", tree.root_node().to_sexp());
+
+    let query = Query::new(
+        elixir_lang,
+        r#"
+(
+    (comment) @comment
+    .
+    [
+        (call target: (identifier) @identifier (arguments (string (quoted_content) @import_file)))
+        (call 
+            target: (identifier)
+          (arguments 
+            (call 
+                target: (_) @file_checking 
+      	        (arguments (string (quoted_content) @checked_file))))
+          (do_block
+			        (call 
+				        target: (identifier) @identifier
+				        (arguments (string (quoted_content) @import_file))))
+		        )
+	    (binary_operator 
+		    left: (call 
+			    target: (_) @file_checking 
+			    (arguments (string (quoted_content) @checked_file)))
+		    right: (call 
+			    target: (identifier) @identifier
+			    (arguments (string (quoted_content) @import_file))))
+    ]
+    (#eq? @file_checking "File.exists?")
+    (#match? @identifier "import_config|import_config!")
+    (#match? @comment "\# wukong.mindvalley.dev/config-secrets-location:")
```

Comment:
we should also match the case of `#wukong.mindvalley.dev/config-secrets-location:`

Reply:
alright, I update the pattern so now it will match `#wukong.mindvalley.dev/config-secrets-location:`